docs/usage/configuration-options.md

```diff
@@ -244,10 +244,27 @@ Renovate also allows users to explicitly configure `baseBranches`, e.g. for use
 
 - You wish Renovate to process only a non-default branch, e.g. `dev`: `"baseBranches": ["dev"]`
 - You have multiple release streams you need Renovate to keep up to date, e.g. in branches `main` and `next`: `"baseBranches": ["main", "next"]`
+- You want to update your main branch and consistently named release branches, e.g. `main` and `release/<version>`: `"baseBranches": ["main", "/^release\\/.*/"]`
 
 It's possible to add this setting into the `renovate.json` file as part of the "Configure Renovate" onboarding PR.
 If so then Renovate will reflect this setting in its description and use package file contents from the custom base branch(es) instead of default.
 
+`baseBranches` supports Regular Expressions that must begin and end with `/`, e.g.:
+
+```json
+{
+  "baseBranches": ["main", "/^release\\/.*/"]
+}
+```
+
+For ignoring certain branches, a special negated regex syntax can be used, e.g.:
```

Comment:
negation makes no sense, because another regex will still work on full list. same behavior like autodiscover filter. we should note that here too.

Reply:
Important observation, thanks!